README.md

```diff
@@ -94,55 +95,77 @@ That should create an executable `target/release/umi-transfer` that can be place
 
 ```shell
 ./target/release/umi-transfer --version
-umi-transfer 1.0.0
+umi-transfer 1.5.0
 ```
 
 ## Usage
 
->### Performance Note
->
->The decompression and compression used within umi-transfer is single-threaded, so to get the most reads per minute performance, see the [high performance guide](#high-performance-guide)
+The tool requires three FastQ files as input. You can manually specify the names and location of the output files with `--out` and `--out2` or the tool will automatically append a `with_UMI` suffix to your input file names. It additionally accepts to choose a custom UMI delimiter with `--delim` and to set the flags `-f`, `-c` and `-z`.
```

Comment:
Did we drop support for single-end mode? That would be only two input fastq:s. Mostly just curious

Reply:
Yes, we dropped that with version 1.0 See the _Discontinued Previous Features_ section in the [_release notes_](https://github.com/SciLifeLab/umi-transfer/releases/tag/v1.0.0). It was too much clutter with the previous code organization, which did not cleanly separate the iterators from the file i/o. 

After the current refactoring, it would actually not be that hard to bring it back or also enable reading one input stream from `stdin`. But I also don't see that a singleton feature would be used frequently. Having paired reads to me seems to be the standard in recent years, and old data has no UMIs.